pydantic/env_settings.py

```diff
@@ -57,10 +57,10 @@ def _build_environ(self) -> Dict[str, Optional[str]]:
         return d
 
     class Config:
-        env_prefix = 'APP_'
+        env_prefix = ''
         validate_all = True
         extra = Extra.forbid
         arbitrary_types_allowed = True
-        case_insensitive = False
+        case_insensitive = True
```

Comment:
`case_insensitive = False` is a double negative.

Let's take this opportunity to change the name to `case_sensitive = False`.